persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java

```diff
@@ -70,11 +70,9 @@ public void testXmlConfig() throws IOException {
 
       Cache<String, String> cache = cacheManager.getCache("testCache");
 
-      cache.put("hello", "there 60 xml");
+      cache.put("hello", "there 52 xml");
       cache.stop();
       cacheManager.stop();
-
-      Util.recursiveFileRemove("/tmp/leveldb/60");
```

Comment:
Please ignore my previous comment. Yes, you are absolutely right... I need glasses :)